frontend/src/components/Header/Header.tsx

```diff
@@ -43,18 +56,20 @@ export const Header = ({ page }: { page: string }) => {
     return (
         <>
             <StyledTopBar>
-                <HandPointer>
-                    <TopBar.Header
-                        onClick={() => {
-                            window.location.href = `${config.FRONTEND_BASE_ROUTE}/FrontPage`
-                        }}
-                    >
-                        <Typography variant="body_long_bold" color="primary">
-                            Flotilla
-                        </Typography>
-                    </TopBar.Header>
-                </HandPointer>
-                <Typography> {installationName}</Typography>
+                <StyledWrapper>
```

Comment:
Have you tested if this new style, or the iphone styles in general, work with the banners? I can try test it, but that might not be until late tomorrow or next week.

Reply:
I honestly haven't, can take a look tomorrow to see how that is going

Reply:
![image](https://github.com/equinor/flotilla/assets/33518988/c277070c-beaa-47a2-bc17-764a97b2235e)
Not bad :)

Reply:
![image](https://github.com/equinor/flotilla/assets/108514046/e4e7c6c7-1ff4-4812-bb91-88656e056ef1)